### ISSUE TITLE
[Suggestion] Add webcam support

### ISSUE THREAD

**Issue by ilovefreesw** [MarconLP/snapify#13]
Along with screen recording if it gives an option to record webcam as well then it would be great.

**Comment by ilovefreesw** [comment]
Can take help from this :https://github.com/pxi-gmbh/pipfy

https://pipfy.pxi.design/webcam.html

@MarconLP

**Comment by theprogrammerinyou** [comment]
Hi, can i take up this task?

**Comment by MarconLP** [comment]
Yeah of course, @theprogrammerinyou! Feel free to give it a shot.

**Comment by theprogrammerinyou** [comment]
Hi @MarconLP can you help me in adding the github OAuth callback URL, not understanding what should that be.

**Comment by MarconLP** [comment]
@theprogrammerinyou for local development you should use `http://localhost:3000/api/auth/callback/github`

**Comment by 9akashnp8** [comment]
hey guys, this is a great feature suggestion. is anyone working on this?

**Comment by theprogrammerinyou** [comment]
Hi, @9akashnp8, yes am currently working on it, but not finding ample time to complete the feature, we can work collectively on it, or i would try to complete it ASAP.

**Comment by 9akashnp8** [comment]
sure, we can work on it together. let me know if you need any help.

**Comment by harshitkumar31** [comment]
I needed this feature and pushed my changes to https://github.com/MarconLP/snapify/pull/30
Feel free to update it.

**Comment by Parajulibkrm** [comment]
Any update on this? @MarconLP
